### DISCUSSION TITLE
Where find site save games download apollo ps4 ?

### DISCUSSION THREAD

**Discussion by ToniFabri** [bucanero/apollo-ps4#134] [Q&A]
Because data base Apollo not has all saves 
and when I find the keystone error, I'm noob in converting the right way looking for the format ready to pass to HD PS4, thanks

**Comment by Masamune3210** [comment]
Google is your friend. "Ps4 save *insert game name here*"

**Comment by bucanero** [comment]
Apollo Database is a community effort. If people using Apollo export, upload and share saves, then more content will be available. 
But the lack of effort in the community means that they don't care or don't want to spend a couple of minutes growing the Database. So I can't fix that. 

You'll have to find saves on your own with google, or try asking on [Discord server](https://discord.com/invite/jmpcCUk), someone might have a save to share.
